### PR TITLE
Show presentation properties in Unity

### PR DESCRIPTION
**Pull request by mgooding**

### CODE REVIEW

File: node/src/IModelHubDownload.ts [hunk 1]
```diff
@@ -3,12 +3,15 @@
...
-import { ElectronAuthorizationBackend } from "@bentley/electron-manager/lib/ElectronBackend";
-import { AuthorizedBackendRequestContext, BriefcaseDb, BriefcaseManager } from "@bentley/imodeljs-backend";
-import { LocalBriefcaseProps } from "@bentley/imodeljs-common";
-import { AccessToken } from "@bentley/itwin-client";
-import { APP_LOGGER_CATEGORY } from "./Main";
+import { Logger } from '@bentley/bentleyjs-core';
+import { ElectronAuthorizationBackend } from '@bentley/electron-manager/lib/ElectronBackend';
+import {
+    AuthorizedBackendRequestContext, BriefcaseDb, BriefcaseManager
+} from '@bentley/imodeljs-backend';
```
Comment (MichaelBelousov) [inline node/src/IModelHubDownload.ts:10]: don't we usually prefer `"` delimiters for our strings in JS? Minor nit, but seems like an unnecessary change here.

**Review by MichaelBelousov — CHANGES_REQUESTED** [review]

File: node/src/IModelHubDownload.ts (hunk 1 above)
Comment (mgooding) [reply node/src/IModelHubDownload.ts]: bad settings from import orderer

**Review by MichaelBelousov — APPROVED** [review]
